CHANGELOG.md

```diff
@@ -1,10 +1,20 @@
 # Changelog
 
-## [Unreleased](https://github.com/ckan/ckanext-dcat/compare/v1.0.0...HEAD)
+## [Unreleased](https://github.com/ckan/ckanext-dcat/compare/v1.1.0...HEAD)
 
 * TBD
 
-## [v1.0.0](https://github.com/ckan/ckanext-dcat/compare/v0.0.9...v1.0.0)
+## [v1.1.0](https://github.com/ckan/ckanext-dcat/compare/v1.0.0...v1.1.0)
+
+* requirements.txt are Python 3 now (#167)
+* Fix `after_show - set_titles` in plugins.py (#172)
+* Add support for DCT.rightsStatement in DCT.accessRights and DCT.rights (#177)
+* Add support for additional vcard representations (#178)
+* Fix format normalization configuration (#175)
+* Introduce the possibility to modify package update/create schema (#176)
+* Python 3 support and new pytest based test suite (#174)
```

Comment:
This is something people will look out for to know what version they can use so can we move it to the top?

Reply:
Do you know to remove the unreleased section for the release tag commit and add the unreleased section in a second commit?

Reply:
Sorry not sure what you mean, I just meant moving the Python 3 support entry to the top of this version changes list. If it's difficult don't worry

Reply:
Ah, ok. Now I understand. 😄 This was misleading because there was not only the line of code you meant.  That's no problem. Nevertheless I will split the commit in two commits.